### DISCUSSION TITLE
Adding a bot that create a build message

### DISCUSSION THREAD

**Discussion by Thomas-Boi** [devicons/devicon#574] [Ideas]
Hey guys,

I have an idea to make a new bot that create a list of PRs that were added since last release. Currently, we have to make the messages by hand (see [this](https://github.com/devicons/devicon/pull/560)) which takes 10-20 minutes depend on how many PR we have. Having a bot would make this faster and more efficient (no missing contributors etc...)

Here is how it would work:
- The bot can be triggered as a GitHub Action (labelling/workflow dispatch/on PR to masater), Node script (NPM cli), or a bash script.
- The bot would use the [GitHub API](https://docs.github.com/en/rest/reference/pulls#list-pull-requests) to get a bunch of pulls. We will restrict the PR to be the ones since last release only. 
- The bot finds the PR name, their links, and their contributors. It would put them together into a nice string that we can copy and paste.
- Depending on how it was triggered, it can print this to the console, comment on the labeled PR, or print to a file.
- Put the string into our build PR

You can try curl/Postman based on the format listed [here](https://docs.github.com/en/rest/reference/pulls#list-pull-requests).

What do you guys think of this? Is this a worthwhile idea?

**Comment by Panquesito7** [comment]
That sounds like an awesome idea. If you need out any help, please tell me. Thanks. 🙂

**Reply by amacado** [reply]
Sure! Would be an improvement for this project! :) looking forward to see the first prototype

**Comment by Thomas-Boi** [comment]
Hey guys,

So the bot is now in PR. Just so you know, there are some limitations:
- Currently, it's only checking the most recent 100 PRs. I think this should be more than sufficient for a release. If needed, we can change it later.
- The GitHub token is needed since GitHub API only allow non-authenticated user to call it 60 times an hour. A token increases it to around a 1000. Thus, if you want to test the code, feel free to remove the authentication header in `get_release_message.py`. The script should still work, as long as it's only checking 60 PRs or less since last release.
